core/conversion/evaluators/aten.cpp

```diff
@@ -270,7 +270,11 @@ auto aten_registrations TORCHTRT_UNUSED =
                  if (tensor_var.isITensor()) {
                    auto tensor = tensor_var.ITensor();
                    if (ctx->input_is_dynamic) {
-                     return dynamic_size_layer(ctx, n, args);
+                     if (ctx->settings.allow_shape_tensors) {
```

Comment:
We should be checking if input is dynamic, shape tensors are enabled **and** the input contains a placeholder dimension before using the shape tensor code path. Otherwise static aten size is sufficient and will cause less errors